src/hotspot/share/utilities/enumIterator.hpp

```diff
@@ -252,6 +266,14 @@ class EnumRange {
     assert(start <= end, "invalid range");
   }
 
+  template <EnumType Start, EnumType End>
```

Comment:
I think this should be using T to match the surrounding code.
```suggestion
  template <T Start, T End>
```

Reply:
BTW, if you are going to change this could you also fix the pre-existing issues with the includes so that it matches the style guide?